## Context
The colony was first established in 1964, in the context of a honeymoon. However, it saw a revival under the guidance of a professor in 1988. At its inception, the colony boasted between 10 and 20 males and an equivalent number of females. Quite early on, this colony bifurcated into two separate locations. It's surmised that both sites likely had the same original founders.

In the late 1990s - early 2000s, new animals were introduced to the colony at least on one occasion. It's pertinent to note that there could exist species within the colony that doesn't pique our interest and was not part of the original group. The likelihood of interbreeding amongst these animals seems scant.

Roughly 5-10 years ago, all animals within the colony were subjected to sampling. Some of these creatures were still alive when the colony was eventually disbanded. Some of the samples are distinctly unique, others correspond to the same animal. Moreover, there were a few animals that lacked identifying bands.

## Sequencing
Regarding the future plans, a single cell line sample should be taken for ultra-long sequencing. Additionally, 6-7 tissues should be utilised for PacBio IsoSeq (talk with Michael Hiller). Plans are afoot to use between 10 and 20 animals for the PacBioReVIO.

A salient point to bear in mind is that the sampling endeavours should be designed to maximise genetic diversity. This is achieved via either pedigree distribution or IBD distribution.

We should investigate if someone is doing the pangenome, and if it is no, push it: https://bat1k.com/.

## Ideas
The split ocurred around 30 years, 30 members (all biologically related, but, in which degree). 

It is expected that the gloss
ophaga soricina bats reach reproductive age at the first year. 

Bats can live up to 5-7 years. Therefore, we are talking about more than 5 generation, being extremelly conservative.

Around 20 years ago new (unrelated?) bats were introduced into the colonies.

Surely, we are in a situation were relatedness should be scare between colonies, and in some cases within. This is consistent with kinship results. But, moreover, we detect some signals of relationship between individuals of different colonies (for example 2nd or 3th degree). This could be caused by:

1- False positives in kinship analyses.

2- Errors in colony assignment.

3- Real relationships between individuals. But, if the colonies were separated too long ago, how they can be related? Here it is where endogamy plays a major role. Just an experiment. This is with humans, but could be translated to bats. 
A pedigree is proposed, emulating the context of the bats, were the founders have several childs, and those childs are splitted before inbreeding. After many generations of autogamy, we analyze the shared identical by descent portions of the chromosome.
We can see how distant relatives, who share a common ancestor beyod 4th generation, are similarly related to double first cousins (somewhat between 2nd and 3rd degree relatives).

![Contextual](https://github.com/MarsicoFL/batPed/assets/55600771/fb7427ff-28f6-4cec-bb54-8691d5bc63b7)


In a more extreme example, consider the following pedigree:
![Captura desde 2023-09-08 10-50-50](https://github.com/MarsicoFL/batPed/assets/55600771/4027ef0d-3a8c-420a-97b4-f749ecaaac8f)

As could be observed, specific members tend to reproduce more than others, due to specific relationships. Here, when IBD is analyzed between descent of different groups sharing a common ancestor beyond 6th generation, we see that the IBD is comparable to half-siblings (2nd degree)

![Captura desde 2023-09-08 10-54-23](https://github.com/MarsicoFL/batPed/assets/55600771/b0fb27cd-c435-463c-bed5-179733d03728)



